src/arch/e2k/entry.c

```diff
@@ -69,8 +79,11 @@ void e2k_entry(struct pt_regs *regs) {
 
 	if (entries_count > 1) {
 		/* XXX currently we support only single core */
-		cpu_idle();
+		//cpu_idle();
```

Comment:
Probably add the corresponding comment here instead of `//cpu_idle`? Smth like "Run cpu_idle on 2nd CPU" and "Run e2k_kernel_start on 1st CPU" below.